rasa/model_training.py

```diff
@@ -51,10 +51,6 @@ class TrainingResult(NamedTuple):
     code: int = 0
 
 
-# backwards compatibility
-train = rasa.train
```

Comment:
Was introduced in [this PR](https://github.com/RasaHQ/rasa/pull/8141). I think that we added that in the PR for backwards compatibility _before_ we actually renamed the file, which made this change a no-op from the beginning. If we want to be a bit extra careful we could consider adding the code below, but I don't think it's necessary:
```suggestion
def train(*args, **kwargs):
    # raise a deprecation warning that this will be removed in 4.x
    return rasa.train(*args, **kwargs)
```